azure-pipelines.yml

```diff
@@ -108,7 +89,7 @@ jobs:
         "lowrisc-toolchain-rv32imcb" \
         "riscv32-unknown-elf" \
         "/tools/riscv" \
-        "-march=rv32imczbb0p92_zbf0p92_zbs0p92_zbt0p92" \
+        "-march=rv32imc_zba0p93_zbb0p93_zbc0p93_zbs0p93" \
```

Comment:
I'm probably missing something obvious here, but what aren't these zb{a,b,c,s}1p0?

Reply:
Support for version 1.0/non-experimental was merged after LLVM 13.0 / 13.0.1.